pkg/cmd/ai/ai.go

```diff
@@ -0,0 +1,25 @@
+package ai
+
+import (
+	"github.com/MakeNowJust/heredoc"
+	"github.com/buildkite/cli/v3/pkg/cmd/factory"
+	"github.com/buildkite/cli/v3/pkg/cmd/validation"
+	"github.com/spf13/cobra"
+)
+
+func NewCmdAI(f *factory.Factory) *cobra.Command {
+	cmd := cobra.Command{
+		Use:   "ai <command>",
+		Short: "Manage AI integration.",
+		Long:  "Work with Buildkite AI.",
+		Example: heredoc.Doc(`
+			# To configure your AI token
+			$ bk ai configure
+		`),
+		PersistentPreRunE: validation.CheckValidConfiguration(f.Config),
```

Comment:
This validation checks an API token and org are configured but its not necessary for this command so can remove it